src/runner.sh

```diff
@@ -127,6 +127,7 @@ function runner::run_test() {
     "$function_name" "$data" 2>&1 1>&3
 
     runner::run_tear_down
+    runner::clean_mocks
```

Comment:
In other frameworks, this function is call `clear_mocks`
![image](https://github.com/TypedDevs/bashunit/assets/6353105/29d95750-9811-4365-a1e4-f3ea49c76635)

Reply:
Done